test_files/jsdoc_types/jsdoc_types.ts

```diff
@@ -4,11 +4,7 @@
  */
 
 import * as module1 from './module1';
-import {ClassOne, value} from './module2';
```

Comment:
Yes. We had to fix a bunch of internal uses of that already, we just didn't upgrade CC for outside use here.